### ISSUE TITLE
The compiler should prohibit overriding only one setter if there is a getter/setter pair in the parent class.

### ISSUE THREAD

**Issue by barkunas** [microsoft/TypeScript#48201]
# Bug Report

### 🔎 Search Terms
The compiler should prohibit overriding only one setter if there is a getter/setter pair in the parent class.

### 🕗 Version & Regression Information

This is the behavior in every version I tried, and I reviewed the FAQ for entries about override getters/setters

### 💻 Code

```ts

class Main {
    protected _aaa: boolean = false
    set aaa(value: boolean) {
        this._aaa = value;
    }
    get aaa(): boolean {
        return this._aaa;
    }
}

class SubMain0 extends Main {
}
class SubMain1 extends Main {
    override set aaa(value: boolean) {
        this.aaa = value;
    }
}
class SubMain2 extends Main {
    override get aaa(): boolean {
        return this._aaa;
    } 
}

console.log(new SubMain0().aaa); // false
console.log(new SubMain1().aaa); // undefined - Bug
console.log(new SubMain2().aaa); // false
```

### 🙁 Actual behavior

After override SubMain class lost "aaa" property 

### 🙂 Expected behavior

SubMain class must use a getter from a parent class
or
The TS compiler should throw an error that a getter/setter pair should override together

**Comment by MartinJohns** [comment]
Sounds like #47581.
